features/howitzer/capybara_screenshot.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Howitzer - Capybara screenshot integration
+
+  Scenario: Test suite can create page screenshot
```

Comment:
I do not like the scenario name. My recommendation:

Scenario: Capybara screenshot library stores screenshot and source page in proper location